project.clj

```diff
@@ -102,7 +102,7 @@
   :lein-ezbake {:vars {:user "puppet"
                        :group "puppet"
                        :build-type "foss"
-                       :java-args ~(str "-Xms2g -Xmx2g -XX:MaxPermSize=256m "
+                       :java-args ~(str "-Xms2g -Xmx2g "
```

Comment:
Makes sense.  Should we also drop it from the higher-level `:jvm-opts` key as well - so we don't have to see the 'perm size not supported' warnings when running from source anymore?